### DISCUSSION TITLE
Start and Stop recording and saving video files to image gallery

### DISCUSSION THREAD

**Discussion by amits26** [monkeymademe/CamUI#98] [General]
Hi, I am very happy to use picamera2-WebUI for one of my hobby project. It has all the capabilities what I was for looking. I wanted to save video stream in image gallery the way it currently does for image capture. I see a function called start_streaming which does the video streaming job but I don't think so it stores the video anywhere. Can you help me if this can be achieved by changing couple of lines in code.

Thanks for such a great work
Amit

**Comment by monkeymademe** [comment]
You are right that function does not save video. There is no saving video function yet. But its something on the roadmap to implement. But I will say its not few lines of code there is a lot going on the background and while ideally I would like the feed recoded it might be a case that the feed gets stopped and then video is recorded. So you might be in a little bit of a wait sorry...
